CalendarApp/NUnitTestProject/EventManagerTests/EventManager.SingleRecurrentEventTest.cs

```diff
@@ -0,0 +1,212 @@
+﻿using System.Collections.Generic;
+using BookingRoom.Helpers;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3.Data;
+using NUnit.Framework;
+
+namespace NUnitTestProject.EventManagerTests
+{
+    [TestFixture]
+    class SignleEventConflictsTest : EventManagerTestClass
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var conflictedEvent = _eventFactory.CreateEvent(
+               new EventTime(2015, 7, 17, 10, 0, 0), new EventTime(2015, 7, 17, 12, 0, 0),
+               "Nunit", "NunitDecription");
+            conflictedEvent.Recurrence = new[] { "RRULE:FREQ=DAILY;COUNT=5" };
+            _meetingBooking.PostEvent(ToEventConverter.ToEvent(conflictedEvent, _timezone), calendarId);
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+            var eventList = _meetingBooking.GetEvents(calendarId);
+            foreach (var e in eventList)
+            {
+                _calendarService.Events.Delete(calendarId, e.Id).Execute();
+            }
+        }
+        [Test]
+        public void EventManager_SingleRecurrent_FindConflicts_InnerPlusOuterIntersection()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var eventForAdd = _eventFactory.CreateEvent(
+               new EventTime(2015, 7, 17, 10, 0, 0), new EventTime(2015, 7, 17, 12, 0, 0),
+               "Nunit", "NunitDecription");
+            eventForAdd.Recurrence = new[] { "RRULE:FREQ=DAILY;COUNT=5" };
+            List<Event> conflictedList = _eventManager.CheckEvent(calendarId, ToEventConverter.ToEvent(eventForAdd, _timezone), false);
+            Assert.AreEqual(conflictedList.Count, 1);
+        }
+        [Test]
+        public void EventManager_SingleRecurrent_FindConflicts_InnerIntersection()
```

Comment:
Consider renaming this method following the pattern I've mentioned previously
Example: CheckEvent_ForEventWithInnerIntersection_ReturnstConflictedEvent